modules/gitops-release-notes-1-15-0.adoc

```diff
@@ -28,10 +28,137 @@ If you have installed the {gitops-title} Operator in the default namespace, run
++
+To enhance security, you must use Kubernetes secrets to manage TLS data. The following fields are introduced in the Argo CD CR to reference Kubernetes secrets of type `kubernetes.io/tls` containing TLS data:
++
+--
+** `.spec.server.route.tls.externalCertificate`
+** `.spec.applicationSet.webhookServer.route.tls.externalCertificate`
+** `.spec.prometheus.route.tls.externalCertificate`
+** `.spec.grafana.route.tls.externalCertificate`
+--
++
+For more information about configuring a route for custom TLS certificate, see examples in link:https://argocd-operator.readthedocs.io/en/latest/usage/routes/#custom-tls-certificates[Custom TLS certificates for Routes]. link:https://issues.redhat.com/browse/GITOPS-5232[GITOPS-5232]
+
+* With this update, Restricted pod security admission (PSA) labels are applied to the `openshift-gitops` namespace to ensure compliance with {OCP} standards. If you are running additional workloads in the `openshift-gitops` namespace, ensure they comply with restricted PSA requirements. If compliance with restricted PSA is not feasible, use a user-defined, cluster-scoped Argo CD instance. In such instances, PSA labels are not applied or controlled by the {gitops-shortname} Operator. link:https://issues.redhat.com/browse/GITOPS-5221[GITOPS-5221]
+
+* Before this update, `trafficmanagement` and `metric` plugins could not be configured by using the `RolloutManager` CR in Argo Rollouts. With this update, you can now specify `trafficmanagement` and `metric` plugins in the `RolloutManager` CR. The {gitops-shortname} Operator automatically updates the config map when these plugin fields are added, modified, or removed. This enhancement allows users to manage `trafficmanagement` and `metric` plugins natively through the `RolloutManager` CR, eliminating the need for manual config map updates. link:https://issues.redhat.com/browse/GITOPS-4217[GITOPS-4217]
+
+* With this update, the *Sources* tab in the Details panel of a multi-source application now includes two new buttons:
++
+--
+** *Add Source*: Opens a new form-based panel for adding a new source entry. You can specify the repository URL and other source-specific details in this panel.
+** *Delete*: Allows you to remove a specific source from the application.
+--
++
+link:https://issues.redhat.com/browse/GITOPS-4175[GITOPS-4175]
+
+* With this update, the {gitops-shortname} Operator has improved tenant isolation by using a separate service account per tenant for the sync operation, instead of using the same service account for both sync and control plane operations. This feature is in alpha state and disabled by default. For more information, see link:https://argo-cd.readthedocs.io/en/latest/proposals/decouple-application-sync-user-using-impersonation/#decouple-application-sync-using-impersonation[Decouple Application Sync using Impersonation]. link:https://issues.redhat.com/browse/GITOPS-3501[GITOPS-3501]
```

Comment:
@Dhruv-Soni11 please remove this feature, we'll be get this probably in 1.16

Reply:
Fixed.